### PR TITLE
Abrt action save container data fix

### PR DESCRIPTION
**Pull request by matejkudera**
Changing  abrt-action-save-container-data location from bin to libexec.

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by jfilak** [comment]
If you change fs location of abrt-action-save-container-data, you should also change fs location of abrt-action-save-package-data.

By the way, proper segregation of changes would be:
- spec file changes
- all other changes

What is the reason for this PR? (don't reply, update Commit messages, please).

**Comment by xsuchy** [comment]
@jfilak I do not get why you wanna split it into more commits? I thought you are in favour of merged commits. And in my opinion this should be done in one commit, because one change without the other will not work. So it should be atomic.

**Comment by jfilak** [comment]
> I thought you are in favour of merged commits.

Your thoughts are wrongs. I am in favor of meaningful commits.

> And in my opinion this should be done in one commit, because one change without the other will not work. So it should be atomic.

True but we were always separating spec file changes from the rest of changes because it is easier to use such commits in downstream repositories (and cherry pick them to RHEL6/7).

I usually go through the history of a project and take a look at old commits before I create new ones.

File: src/daemon/Makefile.am [hunk 1]
```diff
@@ -7,16 +7,18 @@ bin_SCRIPTS = \
     abrt-handle-upload
 
 bin_PROGRAMS = \
-    abrt-action-save-package-data \
```
Comment (jfilak) [inline src/daemon/Makefile.am]: You have removed `abrt-action-save-package-data` from `bin_PROGRAMS` but you haven't added it to `libexec_PROGRAMS`

**Pull request merged**